src/components/00-materials/README.md

```diff
@@ -88,13 +88,30 @@ npm install @svgr/webpack --save-dev
 
 ## Contribution
 
-### Process of adding a new icon/image
+### Process of adding a new image
 
 1. Involve a designer by sending him the new SVG in question, unless the designer already has the file. He will review it and possibly change some things, for example by adjusting inner padding etc. in order to guarantee overall quality.
-2. Remove unnecessary code inside the SVG:
+1. Remove unnecessary code inside the SVG:
    - a. `data-name="Layer 2"` # layer info for vector drawing software
```

Comment:
Pleas use `*` for a. & b. 
<img width="232" alt="Screenshot 2020-08-26 at 10 09 59" src="https://user-images.githubusercontent.com/38284563/91278353-69814400-e784-11ea-879a-d90ab5029403.png">